build.sh

```diff
@@ -78,19 +78,19 @@ if [ $(($(version "$(R --version | awk 'NR==1 {print $3}')") <= $(version "3.3.1
 fi
 
 # Install R changepoint package.
-mkdir -p ${R_LIB_DIR}
-echo "install.packages('devtools', lib='${R_LIB_DIR}', repos='http://cran.us.r-project.org')" | R_LIBS_USER=${R_LIB_DIR} R --no-save
-echo "install.packages('MultinomialCI', lib='${R_LIB_DIR}', repos='http://cran.us.r-project.org')" | R_LIBS_USER=${R_LIB_DIR} R --no-save
-echo "devtools::install_git('git://github.com/rkillick/changepoint', branch = 'master')" | R_LIBS_USER=${R_LIB_DIR} R --no-save
+mkdir -p ${R_LIB_DIR} || exit $?
+echo "install.packages('devtools', lib='${R_LIB_DIR}', repos='http://cran.us.r-project.org')" | R_LIBS_USER=${R_LIB_DIR} R --no-save || exit $?
+echo "install.packages('MultinomialCI', lib='${R_LIB_DIR}', repos='http://cran.us.r-project.org')" | R_LIBS_USER=${R_LIB_DIR} R --no-save || exit $?
+echo "devtools::install_git('git://github.com/rkillick/changepoint', branch = 'master')" | R_LIBS_USER=${R_LIB_DIR} R --no-save || exit $?
 
 # Install the rpy2 Python package.
-which lsb_release > /dev/null 2>&1
+which lsb_release > /dev/null 2>&1 || exit $?
```

Comment:
And has to be remove here (otherwise it'll fail on systems where `lsb_release` doesn't exist, negating the test.